.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Build and Test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Cache Memgraph Docker image
+      id: cache-memgraph-docker
+      uses: actions/cache@v1
+      with:
+        path: ~/memgraph
+        key: cache-memgraph-v1.3.0-docker-image
+    - name: Download Memgraph Docker image
+      if: steps.cache-memgraph-docker.outputs.cache-hit != 'true'
+      env:
+        MG_VERSION: '1.3.0'
+      run: |
+        mkdir ~/memgraph
+        curl -L https://memgraph.com/download/memgraph/v$MG_VERSION/docker/memgraph-$MG_VERSION-community-docker.tar.gz > ~/memgraph/memgraph-docker.tar.gz
+    - name: Load Memgraph Docker image
+      run: |
+        docker load -i ~/memgraph/memgraph-docker.tar.gz
+    - name: Build project
+      run: |
+        docker build -t memgraph:mage .
+    - name: Run Memgraph MAGE image
+      run: |
+        docker run -d -p 7687:7687 memgraph:mage --telemetry-enabled=False
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.7'
+    - name: Install dependencies
+      run:
+        python -m pip install --upgrade pip
+        pip install black neo4j==1.7.3 pyyaml pytest flake8
+        pip install pytest-pylint pytest-timeout pytest-cov pytest-benchmark pytest-flake8 pytest-black
+        pip install -r ./python/requirements.txt
```

Comment:
Shouldn't this be all in one file/script? In general, having a script is nice because then you can run the same locally + use the same script on here.

Reply:
These are only testing requirements, therefore they are going to be added to the python tests directory.